views/menu.html

```diff
@@ -23,6 +23,7 @@
       {{ buttons.disabled('Submitted', { action: 'workflow-submitted' }) }}
       {{ buttons.inGroup('Submit', { action: 'workflow-submit' }) }}
       {{ buttons.inGroup('Commit', { action: 'workflow-commit' }) }}
+      {{ buttons.inGroup('Revert', { action: 'workflow-revert' }) }}
```

Comment:
This is superfluous/unimplemented, yes? It wouldn't know what commit id to revert to. Please remove it. A UX discussion of whether it's too hard to find the history feature is always welcome, but it's outside our scope for this particular ticket.

Reply:
Yes, this was a false start from my initial interpretation of the requirements - add a revert button to the in-context menu... still think it would be a nice UX :)

done